app/ui/parse_resume.py

```diff
@@ -51,6 +52,32 @@ def __init__(self,
         self.hard_skills = set(hard_skills)
         self.nlp = spacy.load('en')
 
+    def get_skills_hard(self, input_text):
+        """
+        Given a string with an input document, return an array
+        of "skills".
+
+        Args:
+            input_text (str): A string representing some document
+                              like a resume
+        """
+
+        # check inputs
+        assert isinstance(input_text, str)
+
+        in_text = input_text
+        for r in [' ', ',', ';', ':', '\n', '!']:
+            if r in in_text:
```

Comment:
This check is not necessary. `replace()` won't fail if the thing isn't found

```
x = "word"
y = x.replace("!", " ")
print(y)
```

Reply:
will change